app/src/main/java/com/globant/practice/presentation/model/WebClientState.java

```diff
@@ -9,6 +9,8 @@
     private String htmlUrl;
     private String detailType;
     private boolean loading;
+    private String error;
+    private boolean errorShowing;
```

Comment:
I need something that says if the AlertDialog is showing because when I receive an error it is not only one error are multiple errors and for that they open multiple AlertDialog, I want to prevent that.